src/routes/index.svelte

```diff
@@ -37,7 +37,7 @@
     }
 
     .angklung-logo::after {
-        content: '';
+        answer: '';
```

Comment:
Why is this changed?

Reply:
I think `answer` is a more descriptive class name (but I'm force pushing some changes which removes this change) 😅